lib/components/Login.js

```diff
@@ -89,35 +83,22 @@ var Login = React.createClass({
 
   handleLogin: function(e) {
     e.preventDefault();
-    var username = this.refs.username.getDOMNode().value;
-    var password = this.refs.password.getDOMNode().value;
-    var remember = this.refs.remember.getDOMNode().checked;
+    var username = this.refs.username.value;
```

Comment:
Yup, in fact this branch builds on *your* branch to do the React 14 update in the uploader. Remember that? It was *ages* ago. You have two commits at the top of this PR ;)